src/azure-cli/setup.py

```diff
@@ -131,7 +131,6 @@
     'azure-synapse-accesscontrol~=0.2.0',
     'azure-synapse-artifacts~=0.3.0',
     'azure-synapse-spark~=0.2.0',
-    'cryptography>=2.3.1,<3.0.0',
```

Comment:
will remove the dependency here impacting extensions etc? since we don't want extension directly depend on azure-cli-core?

Reply:
`azure-cli` will always install `azure-cli-core`, so moving the dependency version specifier to `azure-cli-core` will have no impact on `azure-cli` and extensions. Previously, `pip install azure-cli-core` individually will fail as `azure-cli-core` indirectly depends on `cryptography` through `adal`, but it's not pinned there.